### ISSUE TITLE
Migrating from Jekyll to Assemble

### ISSUE THREAD

**Issue by jonschlinkert** [assemble/assemble.io#99]
> It would helpful to have a simple migration path for project currently using Jekyll to just be able to npm install assemble and have a few simple steps to convert the project to Assemble.
> Not sure if this would just require a "Jekyll" engine option, but I thought I'd ask if this was a use case you had considered.

See: https://github.com/assemble/assemble/issues/361

**Comment by pburtchaell** [comment]
@hariadi @jonschlinkert @nschonni I noticed you all were on the original issue. Is this still something  that you think we should do?

**Comment by jonschlinkert** [comment]
yeah definitely. I would be great to have at least a brief migration guide. A plugin would be nice too, I have something I did a while back we could use for converting liquid but I'd definitely need to update it.
